lib/indexer/index_documents.rb

```diff
@@ -3,20 +3,13 @@
 
 module Indexer
   class IndexDocuments
-    MIGRATED_TAGGING_APPS = %w{
-      businesssupportfinder
-      calculators
-      calendars
-      collections-publisher
-      contacts
-      contacts-admin
-      hmrc-manuals-api
-      licencefinder
-      policy-publisher
-      smartanswers
-      tariff
-      travel-advice-publisher
-    }
+    NON_MIGRATED_APPS = %w(
+      publisher
+      service-manual-publisher
+      specialist-publisher
+      whitehall
+      non-migrated-app
```

Comment:
It is used in content-tagger to provide a dummy to use in the tests, so you don't have to specify an _actual_ unmigrated app, like you're doing here: https://github.com/alphagov/rummager/pull/613/files?diff=unified#diff-b9b8b16fea186bffa8dcaec1391a4fafR72.

If you use `non-migrated-app` there, we will be able to remove publisher from the list and have the tests still work.